bin/Tiltfile

```diff
@@ -0,0 +1,71 @@
+load('ext://uibutton', 'cmd_button', 'bool_input', 'location')
```

Comment:
I will @rouja review this file.

Reply:
He imports some libs.

Reply:
It's for cmd_button "Run database migration" for instance